### DISCUSSION TITLE
How to properly incorporate download from Google Drive??

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5103] [Help and Questions] [answered]
Hello!

How to incorporate file downloading from Google Drive.??  The problem is that the files on Google Drive and similar hosting services  work a little bit differently and they do not provide direct access / link to the downloaded file, they just provide the ID, which looks something like this.. `https://drive.google.com/u/0/uc?id=18tnd1qL35WRWyGvwheSydxkJF5L0E8-k&export=download` , even if i try to append `#/dl.7z` or something similar it doesnt work..

Is there any way to do it?

**Comment by rashil2000** [comment, marked as answer]
It's not possible to do it on commandline. It might work for small files, but in general you need a special tool to download stuff from Google Drive. See https://stackoverflow.com/questions/25010369/wget-curl-large-file-from-google-drive

**Reply by anderlli0053** [reply]
> It's not possible to do it on commandline. It might work for small files, but in general you need a special tool to download stuff from Google Drive. See https://stackoverflow.com/questions/25010369/wget-curl-large-file-from-google-drive

Yeah, i need it from the command line, so then that is not possible by Scoop, because the Drive cannot provide direct access link to the file, thank you for the help
